Python_Scripts/plot_SolAlg_Supp1.py

```diff
@@ -143,8 +166,14 @@
 # plot legend manually
 ax.plot(0.4, 25000, 'o', fillstyle='full', c='orange', markersize=marker_size)
 ax.plot(0.4, 12500, 'o', fillstyle='full', c='blue', markersize=marker_size)
-plt.text(0.6, 20000, 'AM faster: ' + str(round(len(adams_bdf_x) / len(adams_tsv_file['t_intern_ms'])*100/len(list_directory_adams), 2)) + ' %', fontsize=labelsize)
-plt.text(0.6, 10000, 'BDF faster: ' + str(round(len(bdf_adams_x)/len(adams_tsv_file['t_intern_ms'])*100/len(list_directory_adams), 2)) + ' %', fontsize=labelsize)
+plt.text(
+    0.6, 20000,
+    'AM faster: ' + str(round(len(adams_bdf_x) / len(adams_tsv_file['t_intern_ms'])*100/len(list_directory_adams), 2)) + ' %',
```

Comment:
irgendwas sagt mir, dass diese Zeile jetzt nicht PEP8-konform ist... :D

Reply:
Yes. Didn't want to make it too ugly on first try. But yes, should introduce variables and shorten